tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -210,8 +210,8 @@ def test_start_and_end_filters_are_returned_in_response(self):
         dashboard.projects.set([Project.objects.create(organization=self.organization)])
 
         response = self.do_request("get", self.url(dashboard.id))
-        assert iso_format(response.data["start"]) == start
-        assert iso_format(response.data["end"]) == end
+        assert response.data["start"].replace(tzinfo=None).isoformat() == start
+        assert response.data["end"].replace(tzinfo=None).isoformat() == end
```

Comment:
this one is kinda interesting actually -- the input is timezoneless and the output gains a timezone -- the original test didn't really show this subtlety